src/components/ChatApp/Settings/PreviewThemeChat.js

```diff
@@ -18,28 +18,23 @@ class PreviewThemeChat extends Component {
   sendMessage = event => {
     event.preventDefault();
     this.addMessage(this.state.message, 'You');
-    let self = this;
-    $.ajax({
-      type: 'GET',
-      url:
-        'https://api.susi.ai/susi/chat.json?q=' +
-        encodeURIComponent(this.state.message),
-      contentType: 'application/json',
-      dataType: 'json',
-      success: function(data) {
-        if (data.answers[0]) {
-          self.addMessage(data.answers[0].actions[0].expression, 'SUSI');
+
+    let message = encodeURIComponent(this.state.message);
+    getSusiPreviewReply(message)
```

Comment:
Call this via action.

Reply:
@akshatnitd Is it necessary to call it via action? This is not changing the state or using reducer.